components/sup/src/manager/mod.rs

```diff
@@ -711,9 +754,11 @@ impl Manager {
                 .expect("Services lock is poisoned!")
                 .values_mut()
             {
+                let service_timer = service_hist.start_timer();
                 if service.tick(&self.census_ring, &self.launcher) {
                     self.gossip_latest_service_rumor(&service);
                 }
+                service_timer.observe_duration();
```

Comment:
When I wrote this code, I didn't know that the time was recorded when the timer object was dropped.  I thought `observe_duration()` was the only way to record the time.  I can switch it to the implicit version, with a comment so that it's clear what's happening.